pallas-network/tests/protocols.rs

```diff
@@ -619,10 +619,10 @@ pub async fn local_state_query_server_and_client_happy_path() {
             let transaction_id = Hash::from(txbytes);
             let index = AnyUInt::MajorByte(2);
             let lovelace = AnyUInt::MajorByte(2);
-            let hex_datum = "9118D81879189F18D81879189F1858181C18C918CF18711866181E185316189118BA";
-            let datum = hex::decode(hex_datum).unwrap().into();
-            let tag = TagWrap::<_, 24>::new(datum);
-            let inline_datum = Some((1_u16, tag));
+            //let hex_datum = "9118D81879189F18D81879189F1858181C18C918CF18711866181E185316189118BA";
+            //let datum = hex::decode(hex_datum).unwrap().into();
+            //let tag = TagWrap::<_, 24>::new(datum);
+            let inline_datum = None;
```

Comment:
It seems that the older code was constructing a `Data` variant of `DatumOption`, but I cannot tell what kind of `PlutusData` it encloses. Perhaps @franciscojoray has some info regarding this.

In any case, this node response is artificial because the size of whole UTxO set is obviously gigantic.